api_tests/nodes/views/test_node_detail.py

```diff
@@ -821,6 +824,38 @@ def test_can_make_project_public_if_admin_contributor(
             project_private.reload()
             assert project_private.is_public
 
+    def test_make_project_private(
+        self, app, url_public, project_public, user
+    ):
+        # If the public node exceeds the the private storage limit
+        key = cache_settings.STORAGE_USAGE_KEY.format(target_id=project_public._id)
+        storage_usage_cache.set(key, 7500000000, settings.STORAGE_USAGE_CACHE_TIMEOUT)
```

Comment:
Probably should get the default storage cap, multiply by the GB setting, and add something to get this rather than using a number. That way, if the default cap changes, the test won't break.